docs/topics/install/help/manifest-parsing.md

```diff
@@ -5,9 +5,12 @@ The installer retrieves information required to install AES over the Internet. S
 ## What's next?
 
 1. Restore Internet connectivity. Perhaps there is a web proxy interfering with access to the Internet.
+
+   Try `curl -ISsf https://www.getambassador.io/` to verify that your computer can reach important websites. This command will show "200" in the first few lines upon success.
```

Comment:
I would prefer that the instructions be even more detailed. Specifically, instead of saying "This command will show 200 in the first few lines upon success", say something like this:
Successfully reaching the internet will show something like this:
HTTP/2 200 
cache-control: public, max-age=0, must-revalidate
content-length: 0
content-type: text/html; charset=UTF-8
...
The exact results are not important as long as the "200" shows up as the return code (which it does in that example). A result something like:
curl: (7) Failed to connect to 167.170.215.127 port 443: Network is unreachable
indicates that there is something preventing you from reaching the Internet.

Something more instructive like that.

Reply:
Yes! Updated:
- aes-crd-manifests.md
- aes-manifests.md
- download-error.md
- manifest-parsing.md